gdal/swig/python/osgeo/ogr.py

```diff
@@ -4864,10 +4864,7 @@ def ExportToJson(self, as_object = False, options = None):
         for key in self.keys():
             fld_defn = self.GetFieldDefnRef(self.GetFieldIndex(key))
             if fld_defn.GetType() == _ogr.OFTInteger and fld_defn.GetSubType() == _ogr.OFSTBoolean:
-                if self.GetField(key):
-                    output['properties'][key] = True
-                else:
-                    output['properties'][key] = False
+                output['properties'][key] = bool(self.GetField(key))
```

Comment:
ah and note that this change should also be done in swig/include/python/ogr_python.i which is the master file